content/2_Bronze/Complete_Rec.mdx

```diff
@@ -330,23 +373,22 @@ $\texttt{perms}$.
 
 ```cpp
 #include <bits/stdc++.h>
-
 using namespace std;
 
 string s;
 vector<string> perms;
 int char_count[26];
 
-void search(string curr) {
-	// we've finished creating a permutation
+void search(const string &curr = "") {
```

Comment:
original was more intuitive bruh

Reply:
my man i changed basically nothing

Reply:
then dont change it

Reply:
ooo references, how scary

Reply:
why add when you could just not

Reply:
it's good practice to pass references when you can
my very first pr on this site taught me that